### ISSUE TITLE
Dataset - download

### ISSUE THREAD

**Issue by ElenaMattei** [ruslanmv/ai-medical-chatbot#2]
May we download the dataset for research purposes only?

**Comment by ruslanmv** [comment]
Yes sure you can download the dataset to do research. The aim of this project is to develop, create and share a free medical chatbot for everyone.

**Comment by ajay-vishnu** [comment]
Hey, I'm getting 404 when I try to access the dataset. Could you please share any other sources? That'd be helpful. Great work by the way.
